docs/en/toolkit/learning-resources/tutorial.md

```diff
@@ -93,15 +93,27 @@ Use Shotgun Desktop (Desktop) to take over your project's configuration. RMB cli
 
 {% include figure src="../../../images/toolkit/learning-resources/tutorial/image_5.png" caption="Select the **Advanced project setup…** in the Desktop popup menu" %}
 
-{% include figure src="../../../images/toolkit/learning-resources/tutorial/image_6.png" caption="Choose the **Shotgun Default** configuration type" %}
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_01.png" caption="Choose the **Shotgun Default** configuration type" %}
 
-{% include figure src="../../../images/toolkit/learning-resources/tutorial/image_7.png" caption="Choose the **Default configuration**" %}
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_02.png" caption="Choose the **Default configuration**" %}
 
-{% include figure src="../../../images/toolkit/learning-resources/tutorial/image_8.png" caption="Enter the name of the folder where your project's files will live." %}
+If this is your first time setting up a Shotgun project, you'll also be prompted to define a storage location for your project data.  Otherwise, you can select an existing storage location.
 
-If this is your first time setting up a Shotgun project, you'll also be prompted to define a storage location for your project data.
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_03.png" caption="Create a new storage." %}
 
-{% include figure src="../../../images/toolkit/learning-resources/tutorial/image_9.png" caption="Make a note of the configuration path you select for the current operating system." %}
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_04.png" caption="Name your new storage.  Remember, this storage is site-wide and not project specific." %}
+
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_05.png" caption="Set the path(s) where this storage will be accessible on the operating systems you intend to use." %}
+
+You can view and edit the storages for your shotgun site in your **Site Preferences**, under the **File Management** section.  You can learn more about these settings [here](https://support.shotgunsoftware.com/hc/en-us/articles/219030938).
+
+Now that you have a storage location selected, you'll choose the name of the directory in that location for your new project.
+
+{% include figure src="../../../images/toolkit/learning-resources/tutorial/wizard_06.png" caption="Enter the name of the folder where your project's files will live." %}
+
+For this tutorial, we'll be using a centralized configuration.  The **Distributed Setup** option provides an alternate option that can provide a different set of benefits, and may be the preferred option for studios without fast shared storage.  You can learn more about the pro and cons of different configuration setups in the [Toolkit Administering](https://www.youtube.com/watch?v=7qZfy7KXXX0&list=PLEOzU2tEw33r4yfX7_WD7anyKrsDpQY2d&index=2) presentation.
```

Comment:
"Toolkit Administration" is the name of the presentation, not "Toolkit Administering".

Reply:
🤦‍♂️ ✅